### PR TITLE
ICON SCM: convert output to DEPHY and do simple plots

### PR DESCRIPTION
**Pull request by martin-koehler**
With those modifications to the COMBLE SCM comparison, I am submitting my ICON-SCM default simulation.  I have add a notebook to convert ICON-SCM format to DEPHY format and another notebook to do the simple check-plots.  I have also done a small correction.

### CODE REVIEW

File: notebooks/conversion_output/convert_WRF_LES_output_to_dephy_format.ipynb
Comment (twjuliano) [inline notebooks/conversion_output/convert_WRF_LES_output_to_dephy_format.ipynb:1]: @martin-koehler Please remove this file from the commit.

File: notebooks/plotting/check_scm_icon.ipynb
Comment (twjuliano) [inline notebooks/plotting/check_scm_icon.ipynb:1]: @martin-koehler I think you can also remove this file. We'll maintain the 'check_scm' master script and update as we start pulling in folks model runs.

File: notebooks/plotting/functions_plotting.py
Comment (twjuliano) [inline notebooks/plotting/functions_plotting.py:1]: @martin-koehler Just wondering whether the changes you made here were to fix a bug or enhance what we had or something else?

Comment (martin-koehler) [reply notebooks/plotting/functions_plotting.py:1]: I had make those fixes for plotting the ICON SCM.  This has something to do with the height coordinates.  We had this discussion, that previously most of your models were using pressure coordinates.  So this change, I remember was necessary.  But obviously, I haven't tested it on all other model output.  So be careful.

File: notebooks/conversion_output/convert_WRF_LES_output_to_dephy_format.ipynb
Comment (martin-koehler) [reply notebooks/conversion_output/convert_WRF_LES_output_to_dephy_format.ipynb:1]: OK, I have reverted this change.  Thanks for pointing out.

File: notebooks/plotting/check_scm_icon.ipynb
Comment (martin-koehler) [reply notebooks/plotting/check_scm_icon.ipynb:1]: OK, I have now removed this file entirely.

**Comment by fridlind** [comment]
I'm sorry for my delay revising the ModelE SCM output format! I've been about halfway through the task for days, and taken off task dealing with other deadlines. I will try to commit the new SCM processing for ModelE3 by COB tomorrow. Then it will have the correct format for the plotting scripts.

From: Martin Köhler ***@***.***>
Reply-To: ARM-Development/comble-mip ***@***.***>
Date: Thursday, May 16, 2024 at 11:00 AM
To: ARM-Development/comble-mip ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [BULK] Re: [ARM-Development/comble-mip] ICON SCM: convert output to DEPHY and do simple plots (PR #51)

CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.



@martin-koehler commented on this pull request.

________________________________

On notebooks/plotting/functions_plotting.py<https://github.com/ARM-Development/comble-mip/pull/51#discussion_r1603543128>:

I had make those fixes for plotting the ICON SCM. This has something to do with the height coordinates. We had this discussion, that previously most of your models were using pressure coordinates. So this change, I remember was necessary. But obviously, I haven't tested it on all other model output. So be careful.

**Review by twjuliano — APPROVED** [review]

**Pull request merged**